core/src/main/scala/org/broadinstitute/dsde/rawls/jobexec/SubmissionMonitorActor.scala

```diff
@@ -393,6 +393,18 @@ trait SubmissionMonitor extends FutureSupport with LazyLogging with RawlsInstrum
         // figure out the updates that need to occur to entities and workspaces
         updatedEntitiesAndWorkspace = attachOutputs(workspace, workflowsWithOutputs, entitiesById, outputExpressionMap)
 
+        // for debugging purposes
+        workspacesToUpdate = updatedEntitiesAndWorkspace.collect { case Left((_, Some(workspace))) => workspace }
+        entityUpdates = updatedEntitiesAndWorkspace.collect { case Left((Some(entityUpdate), _)) if entityUpdate.upserts.nonEmpty => entityUpdate }
+        _ = if (workspacesToUpdate.nonEmpty && entityUpdates.nonEmpty)
+              logger.info("handleOutputs writing to both workspace and entity attributes")
+            else if (workspacesToUpdate.nonEmpty)
+              logger.info("handleOutputs writing to workspace attributes only")
+            else if (entityUpdates.nonEmpty)
+              logger.info("handleOutputs writing to entity attributes only")
+            else
+              logger.debug("handleOutputs writing to neither workspace nor entity attributes; could be errors")
```

Comment:
Should be consistent? Or maybe even `error` since it's a possible error?
```suggestion
              logger.info("handleOutputs writing to neither workspace nor entity attributes; could be errors")
```

Reply:
my goal here was to not spam logs too much - I believe that errors are logged elsewhere, so I didn't want to add anything here

Reply:
buuut ... I think you're right, I'll make it info - even in the case where it doesn't write to entities or workspaces, it will still write to the db to save errors, that's good to track too